realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -575,6 +575,7 @@ public void deleteAll() {
      * Deletes the Realm file defined by the given configuration.
      */
     static boolean deleteRealm(final RealmConfiguration configuration) {
+        final String management = ".management";
```

Comment:
Yes, we skipped a few core releases. Of course I should have fixed with in another PR.